src/utils/index.js

```diff
@@ -294,15 +294,52 @@ export const doesPlotContainCrop = plot =>
 
 export const getLifeStageRange = memoize((
   /** @type {farmhand.cropTimetable} */ cropTimetable
-) =>
-  [SEED, GROWING].reduce(
+) => {
+  return [SEED, GROWING].reduce(
     /**
      * @param {farmhand.cropLifeStage[]} acc
      */
-    (acc, stage) => acc.concat(Array(cropTimetable[stage]).fill(stage)),
+    (acc, stage) => {
+      if (Array.isArray(cropTimetable[stage])) {
+        return acc.concat(
+          cropTimetable[stage].reduce(
+            (acc2, value) => acc2.concat(Array(value).fill(stage)),
+            []
+          )
+        )
+      }
```

Comment:
This new logic needs test coverage as well: https://github.com/jeremyckahn/farmhand/blob/cfdab66a5347427e16310e8e65b0484b08a08a1b/src/utils/index.test.js#L529-L537

Reply:
updated